twitter_ads/http.py

```diff
@@ -141,6 +141,8 @@ def __init__(self, code, headers, **kwargs):
             raw_response_body = self._raw_body
 
         try:
+            if not isinstance(raw_response_body, str):
```

Comment:
I'm not entirely sure if that's the case, I'll test it out and get back to you on that.